HW_1/task_2.py

```diff
@@ -0,0 +1,21 @@
+# 2.Треугольник существует только тогда, когда сумма любых двух его сторон больше третьей. 
+# Дано a, b, c - стороны предполагаемого треугольника. 
+# Требуется сравнить длину каждого отрезка-стороны с суммой двух других. 
+# Если хотя бы в одном случае отрезок окажется больше суммы двух других, 
+# то треугольника с такими сторонами не существует. 
+# Отдельно сообщить является ли треугольник разносторонним, равнобедренным или равносторонним.
+
+a = int(input('Введите длину стороны a: '))
+b = int(input('Введите длину стороны b: '))
+c = int(input('Введите длину стороны c: '))
+
+if a + b > c and a + c > b and b + c > a:
+    print('Треугольник существует')
+    if a == b == c:
+        print('Треугольник равносторонний')
+    elif a == b or a == c or b == c:
+        print('Треугольник равнобедренный')
+    else:
+        print('Треугольник разносторонний')
```

Comment:
Могу еще предложить такой вариант
```suggestion
    elif a != b != c:
        print('Треугольник разносторонний')
    else:
        print('Треугольник равнобедренный')
```